src/vue/i18n.js

```diff
@@ -0,0 +1,18 @@
+import Vue from "../../node_modules/vue/dist/vue.esm.browser.js";
+import VueI18n from "../../node_modules/vue-i18n/dist/vue-i18n.esm.browser.js";
+
+Vue.use(VueI18n);
+
+const vueI18n = new VueI18n(i18next);
+
+i18next.on("initialized", () => {
+    vueI18n.setLocaleMessage("en", i18next.getDataByLanguage("en").messages);
+});
```

Comment:
Introducing this vue i18n for translations will complicate things a little... we don't use the `i18next` library directly because we have first a `chrome app` and we use the `chrome` way for pass parameters in the messages file, was positional, with arrays, etc. 
If the vue i18n give us benefits is not a problem, but maybe we need to modify some translations when we add it to the vue way.

Reply:
There exists vue i18next version as well. But I've chose this one because other one currently doesn't have the ESM compliant build.

The main reason to use this is to get translations like `$t("versionLabelConfigurator.message")` in the Vue templates. `i18next` seems to have decent event system implemented so it's quite easy to make them two talk. Eventually this would need the refactor of course. But I think that would be more beneficial to do once all of the UI is in Vue( or well any other framework.

Reply:
What I say is that precisely we use i18nex but with some "wrapper" to maintain compatibility with the old chrome way for some of the old messages that use it:
https://github.com/betaflight/betaflight-configurator/blob/59076c0c3aba5e83e0ae47edebb5f0780215298f/src/js/localization.js#L62-L86

Not a big problem, but we need to take it into account.